### PR TITLE
feat(node/http): upgrade event

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

File: node/internal/child_process.ts [hunk 1]
```diff
@@ -324,6 +324,27 @@ function keys<T extends Record<string, unknown>>(object: T): Array<keyof T> {
...
+      if (os.signals[name] === signal) {
+        return name as Deno.Signal;
+      }
+    }
+    throw new ERR_UNKNOWN_SIGNAL(String(signal));
+  }
+
+  const denoSignal = signal as Deno.Signal;
+  if (os.signals[denoSignal] != null) {
+    return denoSignal;
```
Comment (bartlomieju) [inline node/internal/child_process.ts:339]: Seems like unrelated change

**Comment by kt3k** [comment]
Can you add any test case?

**Comment by bartlomieju** [comment]
Closing in favor of https://github.com/denoland/deno_std/pull/2457
